wgsl/index.bs

```diff
@@ -9959,50 +9761,6 @@ The following are reserved words:
 
```

Comment:
nil, null, nullptr

Reply:
Sorry, are you asking for these to be reserved?  They already are.

(nil is one of only 5 keywords in Smalltalk)

Reply:
I don't know what I was asking, heh.